packages/cli/src/commands/verify/action.ts

```diff
@@ -10,7 +10,7 @@ export async function action(params: Options & Args & { dontExitProcess: boolean
 
   const controller = new NetworkController(params.network, params.txParams, params.networkFile);
 
-  controller.checkLocalContractDeployed(params.contract, true);
+  controller.throwOrLogErrorForContract(params.contract, true);
```

Comment:
Yeah, I am dissatisfied with `throwOrLogErrorForContract` as well. It can be simplified to `logErrorForContract` because a parameter `throwIfFail` implies that it also can throw.
Funny thing. I only now realized that `checkLocalContractDeployed` stands for check if a contract is local and deployed. It is confusing because it is missing `if/is` parts. The method called `checkIfContractIsLocalOrDeployed` would read much better. To be precise this method checks for three things:
* Contract is in ProjectFile (is local)
* Contract is in NetworkFile (is deployed)
* Deployed contract is up to date `currentBytecode === localBytecode;`
And the method name is missing the third part. I also think using the term `local` for being present in ProjectFile is not the best because if it is local, what is a global contract then? Or remote?
My new take on that is `logErrorIfContractChangedOrNotDeployed`. I know it sounds long but I still have fresh memories of how confusing these methods names can be. If there are many of them it becomes really hard to build a mental map of a project.
It is certainly quite an opinionated topic so I value input on that issue. Let me know what you think about `logErrorIfContractChangedOrNotDeployed`.

Reply:
Maybe `logErrorIfContractNotDeployedLatest`? I'd try to go for the positive: if a contract passes this check, it means it has been deployed and is up to date.